langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -119,6 +157,12 @@ public isolated function getName(Element elem) returns string = @java:Method {
 
 # Changes the name of an XML element.
 #
+# ```ballerina
+# xml:Element s = xml `<person>John</person>`;
+# s.setName("student");
+# s ⇒ <student>John</student>
```

Comment:
How did we chose variable names? `s` here and `p` for the previous function?

Reply:
Was focusing on the outcome `<student>...</student>`. 
Updated in https://github.com/ballerina-platform/ballerina-lang/commit/503e179a741f518021380c7714f358de1a1ea5ef as per the discussion at https://github.com/ballerina-platform/ballerina-lang/pull/38981#discussion_r1062154370